services/ux-backend/main.go

```diff
@@ -91,6 +98,9 @@ func main() {
 	http.HandleFunc("/rotate-keys", func(w http.ResponseWriter, r *http.Request) {
 		rotatekeys.HandleMessage(w, r, cl)
 	})
+	http.HandleFunc("/cleanup-consumer", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
There is no resource named `clenup-consumer`. The name for the resource should make sense:
`/storageconsumers/<consumer_id>`